pennylane/qchem/convert_openfermion.py

```diff
@@ -13,19 +13,90 @@
+from typing import Union
+
 # pylint: disable= import-outside-toplevel,no-member,unused-import
-from pennylane.fermi import FermiSentence, FermiWord
+import pennylane as qml
+from pennylane import numpy as np
+from pennylane.fermi.fermionic import FermiSentence, FermiWord
+from pennylane.ops import LinearCombination, Sum
+from pennylane.qchem.convert import (
+    _openfermion_to_pennylane,
+    _pennylane_to_openfermion,
+    _process_wires,
+)
+from pennylane.wires import Wires
+
+
+def _import_of():
+    """Import openfermion."""
+    try:
+        # pylint: disable=import-outside-toplevel, unused-import, multiple-imports
+        import openfermion
+    except ImportError as Error:
+        raise ImportError(
+            "This feature requires openfermion. "
+            "It can be installed with: pip install openfermion."
+        ) from Error
+
+    return openfermion
+
+
+def from_openfermion_qubit(of_op, tol=1.0e-16, **kwargs):
+    r"""Convert OpenFermion ``QubitOperator`` to a :class:`~.LinearCombination` object in PennyLane representing a linear combination of qubit operators.
+
+    Args:
+        of_op (QubitOperator): fermionic-to-qubit transformed operator in terms of
+            Pauli matrices
+        wires (Wires, list, tuple, dict): Custom wire mapping used to convert the qubit operator
+            to an observable terms measurable in a PennyLane ansatz.
+            For types Wires/list/tuple, each item in the iterable represents a wire label
+            corresponding to the qubit number equal to its index.
+            For type dict, only int-keyed dict (for qubit-to-wire conversion) is accepted.
+            If None, will use identity map (e.g. 0->0, 1->1, ...).
+        tol (float): tolerance value to decide whether the imaginary part of the coefficients is retained
+        return_sum (bool): flag indicating whether a ``Sum`` object is returned
+
+    Returns:
+        (pennylane.ops.Sum, pennylane.ops.LinearCombination): a linear combination of Pauli words
+
+    **Example**
+
+    >>> q_op = QubitOperator('X0', 1.2) + QubitOperator('Z1', 2.4)
+    >>> from_openfermion_qubit(q_op)
+    1.2 * X(0) + 2.4 * Z(1)
+
+    >>> from openfermion import FermionOperator
+    >>> of_op = 0.5 * FermionOperator('0^ 2') + FermionOperator('0 2^')
+    >>> pl_op = from_openfermion_qubit(of_op)
+    >>> print(pl_op)
+        0.5 * a⁺(0) a(2)
+        + 1.0 * a(0) a⁺(2)
+    """
+    coeffs, pl_ops = _openfermion_to_pennylane(of_op, tol=tol)
+    pl_term = qml.ops.LinearCombination(coeffs, pl_ops)
 
+    if "format" in kwargs:
+        if kwargs["format"] == "Sum":
+            return qml.dot(*pl_term.terms())
+        if kwargs["format"] != "LinearCombination":
+            f = kwargs["format"]
+            raise ValueError(f"format must be a Sum or LinearCombination, got: {f}.")
 
-def from_openfermion(openfermion_op, tol=1e-16):
+    return pl_term
+
+
+def from_openfermion_fermionic(of_op, tol=1e-16):
```

Comment:
Please keep this function as it currently exists in master, without changing it.